fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/util/FHIRTermGraphUtil.java

```diff
@@ -72,33 +26,4 @@ public static void setRootLoggerLevel(Level level) {
         Logger rootLogger = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
```

Comment:
Since this is the only code left in the class, maybe it makes sense to inline it where it is used, or rename this class.

Reply:
I anticipate the potential for other methods in the class in the future despite the fact that it is currently a bit sparse.